sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1210,20 +1236,32 @@ def process(self, element, *schema_side_inputs):
       return self._flush_all_batches()
 
   def finish_bundle(self):
-    if BigQueryWriteFn.LATENCY_LOGGING_LOCK.acquire(False):
+    if BigQueryWriteFn.STREAMING_API_LOGGING_LOCK.acquire(False):
```

Comment:
Can all of this be encapsulated entirely in a new class? or at least a helper that can be called in finish_bundle

Ideally we can instrument this without  adding so much to the original implementation.

Reply:
splitted into a separate helper method.